docs/sources/datasources/prometheus.md

```diff
@@ -153,7 +151,14 @@ Regex:
 
 > **Note:** Available in Grafana 7.2 and above
 
-The `$__rate_interval` variable is meant to be used in the rate function. It is defined as max( `$__interval` + _Scrape interval_, 4 \* _Scrape interval_), where _Scrape interval_ is the Min step setting (AKA query*interval, a setting per PromQL query), if any is set, and otherwise the \_Scrape interval* as set in the Prometheus data source (but ignoring any Min interval setting in the panel, because the latter is modified by the resolution setting).
+`$__rate_interval` is the recommended interval to use in the `rate` and `increase` functions. It will "just work" in most cases, avoiding most of the pitfalls that can occur when using a fixed interval or `$__interval`.
+
+```
+OK:       rate(http_requests_total[5m])
+Better:   rate(http_requests_total[$__rate_interval])
+```
+
+Details: `$__rate_interval` is defined as max( `$__interval` + _Scrape interval_, 4 \* _Scrape interval_), where _Scrape interval_ is the Min step setting (AKA query_interval, a setting per PromQL query) if any is set. Otherwise, the _Scrape interval_ is set in the Prometheus data source (but ignoring any Min interval setting in the panel, because the latter is modified by the resolution setting). [This article](https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/) contians additional details.
```

Comment:
contians → contains

The way the original paragraph is cut into two sentences is now a bit misleading. The part about ignoring Min interval now appears to only apply to the Scrape interval setting in the data source, but it applies to the whole thing.

Suggestion:

```
Details: `$__rate_interval` is defined as max( `$__interval` + _Scrape interval_, 4 \* _Scrape interval_), where _Scrape interval_ is the Min step setting (AKA query_interval, a setting per PromQL query) if any is set. Otherwise, the Scrape interval setting in the Prometheus data source is used. (Note that the Min interval setting in the panel is modified by the resolution setting and therefore doesn't have any effect on _Scrape interval_.) [This article](https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/) contains additional details.
```

Reply:
Thanks for the suggestions, I updated the PR.